src/main/java/org/spongepowered/common/mixin/api/mcp/entity/monster/AbstractRaiderEntityMixin_API.java

```diff
@@ -0,0 +1,20 @@
+package org.spongepowered.common.mixin.api.mcp.entity.monster;
+
+import java.util.Optional;
+
+import net.minecraft.entity.monster.AbstractRaiderEntity;
+import org.spongepowered.api.entity.living.monster.raider.Raider;
+import org.spongepowered.api.raid.Raid;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+@Mixin(AbstractRaiderEntity.class)
+public abstract class AbstractRaiderEntityMixin_API extends PatrollerEntityMixin_API implements Raider {
+
+    @Shadow public abstract net.minecraft.world.raid.Raid shadow$getRaid();
+
+    public Optional<Raid> getRaid() {
```

Comment:
If this is implementing the method, please mark it as `@Override`, don't want to mistake on a cursory glance that it's a random added method in a mixin.